server/src/main/java/org/opensearch/search/profile/query/ConcurrentQueryProfileBreakdown.java

```diff
@@ -23,7 +28,21 @@
  * @opensearch.internal
  */
 public final class ConcurrentQueryProfileBreakdown extends ContextualProfileBreakdown<QueryTimingType> {
-    private final Map<Object, AbstractProfileBreakdown<QueryTimingType>> contexts = new ConcurrentHashMap<>();
+    static final String SLICE_END_TIME_SUFFIX = "_slice_end_time";
```

Comment:
Please make those `private`, in tests we could use their expected string values

Reply:
these are used in tests to query the value of keys with these prefix/suffix and QueryTimingType from the returned breakdownMap instead of redefining in test class again. Example [here](https://github.com/opensearch-project/OpenSearch/pull/9248/files#diff-be708772a9b2f0bbeb4a17ed48902b362b503801d91bee36753cea35602bcf68R75). Is there any issue with keeping it package private ?

Reply:
not really an issue, just a suggestion (in tests we could just use "_slice_end_time" directly)

Reply:
I was trying to avoid using explicit names in test such that if we change it in source, tests doesn't need to be updated explicitly. I am leaning to keep it as is.